config/locales/pt-BR.yml

```diff
@@ -145,6 +145,8 @@ pt-BR:
   rule_modified: Regra adicionada
   rule_deleted: Regra deletada
 
+# private_messages/_message
+  private_message_at: postado em -
```

Comment:
You can change it to the following to make it work correctly:
`private_message_sent_at: '%{email} postado em %{at}'`
*maybe this is invalid Brazilian Portuguese* ;)